Assets/MixedRealityToolkit.Extensions/Sharing/Localization/SpatialCoordinateServiceBase.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.Sharing
+{
+    /// <summary>
+    /// Helper base class for <see cref="ISpatialCoordinateService"/> implementations.
```

Comment:
>ISpatialCoordinateService [](start = 41, length = 25)

nit: you may already be doing this, but with these new crefs we should run docfx generation locally and fix any warnings. I think this resolves correctly, but its worth confirming.

When we move to pr's against mrtk_development, docfx generation should be a build step. But right now there is no build infrastructure for our feature/spectatorView.

Reply:
I use VS's intellisense filling these in, so should resolve correctly.

---
In reply to: [270512318](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3732#discussion_r270512318) [](ancestors = 270512318)